docs/design/alias-configuration.md

```diff
@@ -27,10 +27,27 @@ This document explains how additional Crypto transaction flows may be used to cr
 
 ### Alias to Public Key Validation
 
-### Support Public Key or Ethereum Address Alas
+- In case of ED25519 the provided alias must match the public key of the account that we want to create
+- In case of the ECDSA the alias must match the public key or be the public address of the ECDSA key provided/set on the account (when the alias is the public address of the ECDSA key and the key is not set on the account we have to create the account with empty public key field and to store the public address as alias)
+- We can use MiscUtils.isSerializedProtoKey logic to validate keys
+- Additionally we can validate with regex for evm_address (if hex encoded string), otherwise if bytes array check its length
+- We can check existing aliases using AliasManager.lookupIdBy method
+- To derive public address from public key we can use recoverAddressFromPubKey from hapi-utils
+
+### Support Public Key or Ethereum Address Alias
+
+- For CryptoTransfer we already support alias public key auto create
+- To support Ethereum address for CryptoTransfer we will need to: 
+  - Change the implementation in AliasResolver.resolveInternalFungible method, it already has some EVM address handling
+  - Change the AutoCreationLogic.create to expect alias that might not be key but a public address
 
 ### Crypto Create Transaction Alias Logic
 
+- Extract alias from CryptoCreateTransactionBody and validate it in validate() method, checking if the alias matches the key or can be derived from the key (for EVM address alias)
+- For the ED25519 key case when creating the account we should set its alias to the value of the key
```

Comment:
We don't want to default set the alias for ED25519 case

Reply:
I am fine with removing the Crypto Create case for ED25519 key alias, but I think currently the auto create with Crypto Transfer is working with ED25519 keys. That's why I assumed that in order to be consistent we will want to have it for Crypto Create as well. What is the reason we don't want to support this?

Reply:
Yes auto-create with cryptoTransfer is the only way today.
Alias are immutable and ED account with alias don't enable any functionality whereas EC keys with alias open up smart contract compatibility. 
There's a migration path for EC but none for ED

Reply:
Removed the ED25519 case.